test/parallel/test-fs-write-sync-without-offset-and-length.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
Most of the same comments apply to this test. You could probably combine the two tests in one file, separated by block scopes.